cpp/data/zotero_cgi/index.html

```diff
@@ -123,7 +124,7 @@ <h1>Zotero Harvesting</h1>
                                 </td>
                                 <td>
                                     {IF all_journal_methods == "RSS"}<a href="#rss" onclick="TryRss('{all_journal_titles}')">try rss</a>{ENDIF}
-                                    {IF all_journal_methods == "DIRECT"}<a href="#rss" onclick="TryDirect('{all_journal_titles}')">try direct</a>{ENDIF}
+                                    {IF all_journal_methods == "DIRECT"}<a href="#direct" onclick="TryDirect('{all_journal_titles}')">try direct</a>{ENDIF}
```

Comment:
We actually don't support DIRECT as a harvester operation in the config file anymore (see the new wiki entry for more details). Journals are either classified as RSS or CRAWL, i.e, if they have a RSS feed, we use the RSS operation. If not, we use the CRAWL operation. The legacy DIRECT can be emulated with the CRAWL operation by setting the crawl depth to zero (which is the default value).

Reply:
OK i will remove it, do the librarians already know this?

Reply:
I doubt it. It's mentioned in the wiki, though.

Reply:
Removed, but i don't know whether there would really be an error if DIRECT is set in zotero_harvester.conf (seems like it is mapped via an array where DIRECT is still registered)